src/krait/utils/update.py

```diff
@@ -116,12 +115,6 @@ def run_update():
     '''
     o = run_pip('show krait')
     if site.USER_SITE is not None and site.USER_SITE in o:
-        subprocess.call([
-            sys.executable,
-            *'-m pip install --user --upgrade krait'.split()
-        ])
+        run_pip('install --user --upgrade krait')
     else:
-        subprocess.call([
-            sys.executable,
-            *'-m pip install --upgrade krait'.split()
-        ])
+        run_pip('install --upgrade krait')
```

Comment:
`run_pip` uses subprocess.check_output, subprocess.call was chosen here to just act as a pass-through while still throwing output out to stdout